barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.hpp

```diff
@@ -97,11 +100,11 @@ template <class DeciderProvingKeys_> class ProtogalaxyProver_ {
      * multiplication of matrices whose columns are polynomials, as well as taking similar linear combinations of the
      * relation parameters.
      */
-    FoldingResult<Flavor> update_target_sum_and_fold(const DeciderProvingKeys_& keys,
```

Comment:
It doesn't make sense for this method to be the one to pack up the accumulator + proof. The accumulator is computed in place in keys[0] and the proof is now extracted and returned in `PG::prove()`.